packages/core/src/components/logo/logo.tsx

```diff
@@ -21,21 +60,51 @@ export class Logo implements ComponentInterface {
   @Prop()
   ribbon?: string;
 
+  /**
+   * Emitted when the logo is clicked (only when logoUrl is set).
+   *
+   */
+  @Event()
+  dsoLogoClick!: EventEmitter<LogoClickEvent>;
+
+  /**
+   * Emitted when the label in the logo is clicked (only when labelUrl is set).
+   *
+   */
+  @Event()
+  dsoLabelClick!: EventEmitter<LogoLabelClickEvent>;
+
+  private handleLogoClick = (e: MouseEvent) => {
+    e.preventDefault();
+    this.dsoLogoClick.emit({ originalEvent: e, url: this.logoUrl!, isModifiedEvent: isModifiedEvent(e) });
+  };
+
+  private handleLabelClick = (e: MouseEvent) => {
+    e.preventDefault();
+    this.dsoLabelClick.emit({ originalEvent: e, url: this.labelUrl!, isModifiedEvent: isModifiedEvent(e) });
+  };
+
   render() {
     return (
       <Host aria-label={["Omgevingsloket", this.label, this.ribbon && `(${this.ribbon})`].filter((s) => !!s).join(" ")}>
-        <svg fill="none" viewBox="0 0 48 48" height="100%" class="logo-target">
-          <path class="outer" d="M26 0a24 24 0 1 0 0 47.9A24 24 0 0 0 24 0Z" />
-          <path class="middle" d="M24 8A16 16 0 0 0 8 24 16 16 0 1 0 24 8Z" />
-          <path class="inner" d="M24 32a8 8 0 0 0 0-16 8 8 0 0 0 0 16Z" />
-        </svg>
-
-        <div class="logo-wordmark">
-          <span class="logo-wordmark-omgevings">Omgevings</span>
-          <span class="logo-wordmark-loket">loket</span>
-        </div>
-
-        {this.label && <span class="logo-label">{this.label}</span>}
+        {this.logoUrl ? (
+          <a href={this.logoUrl} onClick={this.handleLogoClick}>
```

Comment:
Hier moet nog iets met de styling gedaan worden, want het de svg en de logo-wordmark zitten niet meer op 1 regel. 
@hansgrimm Zou je jouw omnipotente HTML/CSS brein hier op los kunnen laten? Thanks.

Reply:
Done. check wel even of de afstand tussen de 2 links overeenkomt met het design. die kan ik niet zo gauw vinden.